.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Calance generic CI-CD
+on:
+  push:
+    tags:
+    - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+    branches:
```

Comment:
remove the build trigger for branches... let us keep it consistent with tags.

Reply:
done